examples/bond/test/failing-tests.js

```diff
@@ -89,7 +88,7 @@ describe("Bond token failing tests", function () {
 			creator: { ...bondTokenCreator.account, name: "bond-token-creator" },
 		}).assetIndex;
 
-		const creationFlags = Object.assign({}, flags);
+		const appDef = Object.assign({}, appStorageConfig);
```

Comment:
```suggestion
		const appDefinition = Object.assign({}, appStorageConfig);
```

I think a longer a bit more explicit names are easier to read